quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/CatalogLoader.java

```diff
@@ -333,7 +334,7 @@ else if (indexTableName.startsWith("\"public\".")) {
 					parser.consume();
 				}
 
-				List<String> groups = parser.consumeGroup('(', ')', ',');
+				List<String> groups = parser.consumeGroup('(', ')', ',').stream().map(String::trim).collect(Collectors.toList());
```

Comment:
List of columns in index returned as (column1, column2, column3)
This caused an index to be made on a forked table on " column2", which does not exist